app/controllers/ConfirmationController.scala

```diff
@@ -42,9 +42,11 @@ class ConfirmationController @Inject()(
 
   def onPageLoad(ern: String, arc: String): Action[AnyContent] =
     authorisedDataRequestWithCachedMovementAsync(ern, arc) { implicit request =>
-      withAnswer(ConfirmationPage) { confirmationDetails =>
-        logger.info("[onPageLoad] Successful Explain Delay confirmation page displayed")
-        Future.successful(Ok(view(confirmationDetails)))
+      withAnswer(ConfirmationPage) { _ =>
```

Comment:
what as the reason for this change?

Reply:
`confirmationDetails` isn't used anywhere so I changed it to `_`, however it doesn't hurt to make sure it's there.

Reply:
Sorry I meant this whole change in general

Reply:
I'm going to make custom visualisations in Kibana to track how many of the various cancellation reasons we have, similar to what we've already got for RoR and explain delay.